gh_pages/_source/tesseract_interface_roscpp_doc.rst

```diff
@@ -0,0 +1,107 @@
+Tesseract Interface: ROSCPP
```

Comment:
It seems like this tutorial really belongs in `tesseract_ros`, not in this repository